doc/indices.sgm

```diff
@@ -68,6 +68,11 @@
             </para>
           </listitem>
         </itemizedlist>
+        <para>
+          GiST index can be used also for fast finding points closest to the given one 
+          when ordering by an expression with the <literal>&lt;-&gt;</literal> operator
+          is used, as shown in an example below.
```

Comment:
Remove "is used" and move the comma to be at the end of "operator" on the previous line.